client/src/dataset/components/HDF5ParamsForm.tsx

```diff
@@ -114,5 +142,19 @@ export default withValidation<DatasetParamsHDF5, DatasetParamsHDF5ForForm, Datas
         sig_shape: parseShapeInCommaSeparatedString(values.sig_shape),
         sync_offset: values.sync_offset,
     }),
+    customValidation: (values, { info }) => {
+        const dsInfo = getInfoItemForDSPath(values.ds_path, info)
+        if (dsInfo === undefined) {
```

Comment:
@sk1p this is one of the guards on `undefined` dsPath

Reply:
The other is inside `onDSPathChange`.

Reply:
:+1: